vite.config.ts

```diff
@@ -0,0 +1,77 @@
+import react from '@vitejs/plugin-react-swc';
+import { defineConfig } from 'vite';
+// import { VitePWA } from 'vite-plugin-pwa';
+import svgr from 'vite-plugin-svgr';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    svgr({ exportAsDefault: true }),
+    react(),
+    // VitePWA({
```

Comment:
Why is this plugin commented out?

Reply:
unfortunately something about that PWA plugin is causing the e2e tests to crash and not run.  It should not impact anything to have it disabled. I'll work on getting it renabled in a followon pr.